.circleci/config.yml

```diff
@@ -118,15 +118,15 @@ jobs:
             .circleci/copy-configs
       - run:
           name: Run integration test
-          command: .circleci/retry.sh 3 ./gradlew connectedDebugAndroidTest --no-daemon
+          command: .circleci/run-integration-tests.sh
       - run:
           name: Store coverage data
           command: |
               .circleci/retry.sh 3 ./gradlew jacocoFullReport --no-daemon
               tmp_file="$(mktemp)"
               curl -s https://codecov.io/bash > "$tmp_file"
               actual_hash="$(cat $tmp_file | cksum | cut -f 1 -d\ )"
-              expected_hash='2175291078'
+              expected_hash='261829867'
```

Comment:
So this might change more than we expected. Should we vendor this? What's the right (and secure) way? @jamesonwilliams

Reply:
Yeah, I had that thought too.  I went ahead and copied it into our repo for now.  The downside is it will probably require manual updates, but not sure of another option that is safe.

Reply:
I made a separate PR for this: https://github.com/aws-amplify/amplify-android/pull/609